page-factory-core/src/main/java/ru/sbtqa/tag/stepdefs/CoreGenericSteps.java

```diff
@@ -11,6 +11,7 @@
 import ru.sbtqa.tag.pagefactory.exceptions.FragmentException;
 import ru.sbtqa.tag.pagefactory.exceptions.PageException;
 import ru.sbtqa.tag.pagefactory.exceptions.PageInitializationException;
+import ru.sbtqa.tag.pagefactory.find.FindUtils;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ru.sbtqa.tag.pagefactory.find.FindUtils'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=133&issues=AWfPt2rC4tUZZzFFj6Hd&open=AWfPt2rC4tUZZzFFj6Hd)

[ISSUE_KEY]: AWfPt2rC4tUZZzFFj6Hd